code_annotations/config_and_tools/sphinx/extensions/settings.py

```diff
@@ -0,0 +1,125 @@
+"""
+Sphinx extension for viewing feature toggle annotations.
```

Comment:
See other discussion of `config` vs `setting`. 
```suggestion
Sphinx extension for viewing (non-toggle) config annotations.
```

Reply:
I think it's fine for this to be called `setting` for now.
```suggestion
Sphinx extension for viewing (non-toggle) settings annotations.
```

Reply:
:ok_hand: